dds/DCPS/RTPS/ParameterListConverter.cpp

```diff
@@ -492,6 +503,14 @@ bool to_param_list(const ParticipantProxy_t& proxy,
     proxy.availableBuiltinEndpoints);
   add_param(param_list, be_param);
 
+  // TLS_TODO: verify it's the right place to handle extended builtin endpoints
```

Comment:
May not be needed since it's done above in line 373

Reply:
This is for 'proxy', but I do not see where secure parameters are processed for proxy.

Reply:
It could be just the first one that it needs. See comments below.